### DISCUSSION TITLE
HYHeiLiZhiTiJ.ttf to MSDF

### DISCUSSION THREAD

**Discussion by zszq** [soimy/msdf-bmfont-xml#80] [Q&A]
In the test case, I changed DIN_CB.ttf to HYHeiLiZhiTiJ.ttf can not generate all fonts, especially the Chinese part, what should I do to convert the whole HYHeiLiZhiTiJ.ttf to msdf, including Chinese?

**Comment by soimy** [comment]
```
 -i, --charset-file <charset>  user-specified charactors from text-file
```
You can use this option to specify charset files, [sample](https://github.com/soimy/msdf-bmfont-xml/tree/master/assets/charset/) included in this repo.
